File: ATG.Sandbox.Model/OrderResultModel.cs
using System;

namespace ATG.Sandbox.Model
{
    public class OrderResultModel
    {
        public int Id { get; set; }
        public string Side { get; set; }
        public int Quantity { get; set; }
        public string Symbol { get; set; }
        public decimal Price { get; set; }
        public string Status { get; set; }
    }
}

File: ATG.Sandbox.Web/ClientApp/app/model/orderResult.model.ts
export class OrderResult {
    id: number;
    side: string;
    quantity: number;
    symbol: string;
    price: number;
    status: string;

    constructor(id: number, side:string, quantity: number, symbol: string, price:number, status:string) {
        this.id = id;
        this.side = side;
        this.quantity = quantity;
        this.symbol = symbol;
        this.price = price;
        this.status = status;
    }

    
}